texmf wrong path macOS

@yegor256 
While debugging macOS runner faced with problem, that if I run `make install` - `install-texlive.sh` failed because it try to run `tlmgr init-usertree` even if `texmf` folder created with all settings inside, because in macos it is just another path 

```bash
+ tlmgr init-usertree

tlmgr: user mode database already set up in
tlmgr:   /Users/timur/Library/texmf/tlpkg/texlive.tlpdb
tlmgr: not overwriting it.
make: *** [install] Error 1
```

It is because it is trying to check `/Users/timur/` path, not `/Users/timur/Library/`

--- a/installs/install-texlive.sh
+++ b/installs/install-texlive.sh
@@ -46,8 +46,14 @@ if ! tlmgr --version >/dev/null 2>&1; then
   fi
 fi
 
-if [ ! -e "${HOME}/texmf" ]; then
-  "${LOCAL}/help/sudo.sh" tlmgr init-usertree
+if "${LOCAL}/help/is-macos.sh"; then
+  if [ ! -e "${HOME}/Library/texmf" ]; then
+    "${LOCAL}/help/sudo.sh" tlmgr init-usertree
+  fi
+elif "${LOCAL}/help/is-linux.sh"; then
+  if [ ! -e "${HOME}/texmf" ]; then
+    "${LOCAL}/help/sudo.sh" tlmgr init-usertree
+  fi
 fi
 "${LOCAL}/help/sudo.sh" tlmgr option repository ctan
 "${LOCAL}/help/sudo.sh" tlmgr --verify-repo=none update --self

